uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -484,6 +464,22 @@ public function upgrade_1018() {
     return false;
   }
 
+  /**
+   * Disables the Case menu items if Tasks&Assignments is enabled
+   *
+   * @return {boolean}
+   */
+  public function upgrade_1019() {
```

Comment:
the difference is that we disabled "Cases" no matter if {{hrcase}} is enabled. I moved it to 1019 so that we didn't have basically two duplicate upgraders, and still the upgrade would run on both new and already-installed extensions